README.md

```diff
@@ -541,6 +541,8 @@ new ssh2.Server({
 });
 ```
 
+* There are two more examples in the [examples](examples) directory. One is a chat
```

Comment:
It's probably best to tweak this to just say `There are more examples in the `examples` directory`, that way more examples can be added without modifying the readme.